test/e2e_node/environment/setup_host.sh

```diff
@@ -42,38 +42,35 @@ EOF
 fi
 
 # For coreos, disable updates
-if $(sudo systemctl status update-engine &>/dev/null); then
+if sudo systemctl status update-engine &>/dev/null; then
   sudo systemctl mask update-engine locksmithd
 fi
 
 # Fixup sudoers require tty
-sudo grep -q "# Defaults    requiretty" /etc/sudoers
-if [ $? -ne 0 ] ; then
+if ! sudo grep -q "# Defaults    requiretty" /etc/sudoers; then
   sudo sed -i 's/Defaults    requiretty/# Defaults    requiretty/' /etc/sudoers
 fi
 
 # Install nsenter for ubuntu images
-cat /etc/*-release | grep "ID=ubuntu"
-if [ $? -eq 0 ]; then
+if cat /etc/*-release | grep "ID=ubuntu"; then
   if ! which nsenter > /dev/null; then
      echo "Do not find nsenter. Install it."
      NSENTER_BUILD_DIR=$(mktemp -d /tmp/nsenter-build-XXXXXX)
-     cd $NSENTER_BUILD_DIR
+     cd "$NSENTER_BUILD_DIR" || exit 1
      curl https://www.kernel.org/pub/linux/utils/util-linux/v2.31/util-linux-2.31.tar.gz | tar -zxf-
      sudo apt-get update
      sudo apt-get --yes install make
      sudo apt-get --yes install gcc
-     cd util-linux-2.31
+     cd util-linux-2.31 || exit 1
```

Comment:
change reason:
```
In ./test/e2e_node/environment/setup_host.sh line 66:
     cd util-linux-2.31
     ^----------------^ SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
```